features/ouders.feature

```diff
@@ -19,236 +19,234 @@ Rule: De ouder in categorie 2 heeft ouderAanduiding "1" en de ouder in categorie
-    | naam            | waarde |
-    | ouderAanduiding | 2      |
-    En heeft de 'ouder' de volgende 'naam' gegevens
-    | naam      | waarde    |
-    | voornamen | Guîllaumé |
+    | naam            | waarde    |
+    | ouderAanduiding | 2         |
+    | naam.voornamen  | Guîllaumé |
 
 @gba
 Rule: de actuele gegevens van ouders worden geleverd
 
   @gba
   Scenario: oudergegevens gecorrigeerd
-    Gegeven de persoon met burgerservicenummer '999996186' heeft een ouder '1' met de volgende gegevens
+    Gegeven de persoon met burgerservicenummer '000007891' heeft een ouder '1' met de volgende gegevens
     | voornamen (02.10)  |
     | Chantal            |
     En de persoon heeft een ouder '2' met de volgende gegevens
-    | voornamen (02.10)  |
-    | Christiaan         |
+    | voornamen (02.10)  | indicatie onjuist (84.10) |
+    | Christiaan         | O                         |
     En de ouder '2' is gecorrigeerd naar de volgende gegevens
     | voornamen (02.10)  |
     | Mark               |
     Als personen wordt gezocht met de volgende parameters
     | naam                | waarde                                       |
     | type                | RaadpleegMetBurgerservicenummer              |
-    | burgerservicenummer | 999996186                                    |
+    | burgerservicenummer | 000007891                                    |
     | fields              | ouders.ouderAanduiding,ouders.naam.voornamen |
     Dan heeft de response een persoon met een 'ouder' met de volgende gegevens
-    | naam            | waarde |
-    | ouderAanduiding | 1      |
-    En heeft de 'ouder' de volgende 'naam' gegevens
-    | naam      | waarde  |
-    | voornamen | Chantal |
+    | naam            | waarde  |
+    | ouderAanduiding | 1       |
+    | naam.voornamen  | Chantal |
     En heeft de persoon een 'ouder' met de volgende gegevens
     | naam            | waarde |
     | ouderAanduiding | 2      |
-    En heeft de 'ouder' de volgende 'naam' gegevens
-    | naam      | waarde    |
-    | voornamen | Mark      |
+    | naam.voornamen  | Mark   |
 
   @gba
   Scenario: geslachtswijziging ouder
-    Gegeven de persoon met burgerservicenummer 555550001 heeft de volgende ouders in de registratie
-    | Categorie | Voornamen (02.10)  | Geslachtsaanduiding (04.10) |
-    | 2         | Noa                | V                           |
-    | 3         | Johanna            | V                           |
-    | 53        | John               | M                           |
+    Gegeven de persoon met burgerservicenummer '000007892' heeft een ouder '1' met de volgende gegevens
+    | voornamen (02.10) | geslachtsaanduiding (04.10) |
+    | Noa               | V                           |
+    En de persoon heeft een ouder '2' met de volgende gegevens
+    | voornamen (02.10)  | geslachtsaanduiding (04.10) |
+    | John               | M                           |
+    En de ouder '2' is gecorrigeerd naar de volgende gegevens
```

Comment:
het scenario is geen correctie, maar een wijziging

Reply:
Ga ik in een nieuwe PR aanpassen.